masterlist.yaml

```
@@ -6647,6 +6647,13 @@ plugins:
       - crc: 0x626BA4C9
         util: 'SSEEdit v4.0.3'
 
+  - name: 'GrahlsOnSolstheim.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/9311/' ]
```

Since the plugin's name is significantly different to the mod's name, it'd be good to add it's name to the link. Otherwise, it looks good to me.

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/9311/'
        name: 'GRAHL - The Ice Troll (Bloodmoon Creature Restoration Project)'
```